### PR TITLE
amp-bind: Support amp-brightcove

### PR DESCRIPTION
**Pull request by choumx**
Partial for #7825.

- Support binding to `amp-brightcove`'s video source attributes.

/to @mister-ben @aghassemi

### CODE REVIEW

**Comment by kmh287** [comment]
Add integration test for bind?

File: extensions/amp-brightcove/0.1/amp-brightcove.js [hunk 1]
```diff
@@ -46,34 +46,59 @@ class AmpBrightcove extends AMP.BaseElement {
...
-    iframe.src = src;
-    this.applyFillContent(iframe);
-    this.element.appendChild(iframe);
-    this.iframe_ = iframe;
-    return this.loadPromise(iframe);
+    return src;
+  }
+
+  /** @override */
+  mutatedAttributesCallback(mutations) {
```
Comment (jridgewell) [inline extensions/amp-brightcove/0.1/amp-brightcove.js]: Does `#mutatedAttributesCallback`'s caller guard against thrown errors?

Comment (choumx) [inline extensions/amp-brightcove/0.1/amp-brightcove.js]: No, why do you think it should?

Comment (jridgewell) [inline extensions/amp-brightcove/0.1/amp-brightcove.js]: Now we have a `user().assert` in the stack, which'll throw if the expression generates an empty string (probably others in the future will assert more complicated checks). Just want to make sure a thrown error here isn't going to break `#mutatedAttributesCallback` calls to other elements (some for loop gets aborted, etc).

Comment (choumx) [inline extensions/amp-brightcove/0.1/amp-brightcove.js]: Mmm, good point. Filed #8137 to track.

**Pull request merged**